mirrord-layer/src/lib.rs

```diff
@@ -97,110 +94,129 @@ fn init() {
         .with(tracing_subscriber::EnvFilter::from_default_env())
         .init();
 
-    debug!("init called");
+    info!("Initializing mirrord-layer!");
+
     let config = Config::init_from_env().unwrap();
+
     let pf = RUNTIME.block_on(pod_api::create_agent(
         &config.impersonated_pod_name,
         &config.impersonated_pod_namespace,
         &config.agent_namespace,
         config.agent_rust_log,
         config.agent_image,
     ));
+
     let (sender, receiver) = channel::<HookMessage>(1000);
     unsafe {
         HOOK_SENDER = Some(sender);
     };
+
     enable_hooks();
+
     RUNTIME.spawn(poll_agent(pf, receiver));
 }
 
+async fn handle_hook_message(
```

Comment:
nitpick: add inline, compiler will probably do it automatically, but a hint in the direction is always good (one call site, this function is separated from caller only to be more readable. 
same for the other handle function.

Reply:
Done.